src/App/component/ProjectAvatar.jsx

```diff
@@ -79,9 +90,53 @@ function ProjectAvatar(props) {
     setAddRepoDialogOpen(true)
   }
 
+  const openDeletionAlert = () => {
```

Comment:
Since the name `Alert` is a little hard to know its actual implementation is `alert(...)`, suggest add `dialog` or `msgBox` ...etc to help others understand what will happened after called this method.
```suggestion
  const openDeletionAlertDialog = () => {
```